lumibot/brokers/broker.py

```diff
@@ -652,6 +652,15 @@ def should_continue(self):
 
     def market_close_time(self):
         return self.utc_to_local(self.market_hours(close=True))
+    
+    # TODO: _trading_days is initialized in StrategyExecutor!!
+    def is_holiday(self):
+        today = pd.Timestamp(self.datetime.date())
+        
+        if today in self._trading_days.index:
+            return False
+        else:
+            return True
```

Comment:
I agree with korbit that it would be better to invert this and name it "is_market_day" or similar.  The inversion of 'in trading_days' isn't really "holiday" because, among other things, it includes weekends which even though those are pretty much always holidays for some people that isn't really what's intended here.